services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from django.db.models import QuerySet
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+
+    if session_date:
+        parsed_date = session_date.split("-")
+        return MovieSession.objects.filter(
+            show_time__year=parsed_date[0],
+            show_time__month=parsed_date[1],
+            show_time__day=parsed_date[2]
```

Comment:
You can filter by session date directly. Use strptime to retrieve datetime object from string

Reply:
tried to use # `datetime.strptime(session_date, '%Y-%m-%d').date()`
unfortunately doesen't work.. i used `show_time__date`